README.md

```diff
@@ -0,0 +1,23 @@
+# Memes Finder Backend
+A search engine for memes
+
+## Web Scalability
+Is web scale? ✅
+
+## System dependencies (until we get docker)
+### Required
+* [Node (exact version in the `.nvmrc` file)](https://yarnpkg.com/lang/en/)
+* [MongoDB](https://www.mongodb.com/)
```

Comment:
Make sure MongoDB is in gold here. Important. Can you make some animations when users hover ?